lib/style.scss

```diff
@@ -53,6 +53,14 @@
   text-overflow: ellipsis;
 }
 
+// Variables
+
+// This is the color for the nav bar and various row bars throughout the app.
+// Override this value for a quick "re-branding".
+// TODO specify this color using a separate configurable variables file?
```

Comment:
I'd at least move these to the top of the file so that they're featured prominently.